Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+@testable import MongoSwift
```

Comment:
similar to the other file, does this need to be `@testable` and do we need to import NIO?

Reply:
same - just copied some unnecessary imports